src/gui/clips/AutomationClipView.cpp

```diff
@@ -314,24 +314,16 @@ void AutomationClipView::paintEvent( QPaintEvent * )
 		// the outValue of the current node). When we have nodes with linear or cubic progression
 		// the value of the end of the shape between the two nodes will be the inValue of
 		// the next node.
-		float nextValue;
-		if( m_clip->progressionType() == AutomationClip::ProgressionType::Discrete )
-		{
-			nextValue = OUTVAL(it);
-		}
-		else
-		{
-			nextValue = INVAL(it + 1);
-		}
+		float nextValue
+			= m_clip->progressionType() == AutomationClip::ProgressionType::Discrete ? OUTVAL(it) : INVAL(it + 1);
```

Comment:
I think I'd prefer this over the next line indented `= ...`
```suggestion
		float nextValue = m_clip->progressionType() == AutomationClip::ProgressionType::Discrete
			? OUTVAL(it)
			: INVAL(it + 1);
```